showkase-processor/src/main/java/com/airbnb/android/showkase/processor/BaseProcessor.kt

```diff
@@ -107,7 +106,11 @@ abstract class BaseProcessor(
         } catch (e: Throwable) {
             // Errors thrown from within KSP can get lost, making the root cause of an issue hidden.
             // This helps to surface all thrown errors.
-            messager.printMessage(Diagnostic.Kind.ERROR, e.stackTraceToString())
+            if (e is ShowkaseProcessorException && e.element != null) {
```

Comment:
i noticed that the showkase logger in place doesn't every actually get used, so error messages haven't been logging the element. i added this to help, but it seems like there needs to be a bigger clean up